README.md

```diff
@@ -298,6 +298,37 @@ path.deploy_file(
 )
 ```
 
+[Deploy artifact by checksum](https://www.jfrog.com/confluence/display/RTF6X/Artifactory+REST+API#ArtifactoryRESTAPI-DeployArtifactbyChecksum): deploy an artifact to the specified destination by checking if the artifact
+content already exists in Artifactory. If Artifactory already contains a user
+readable artifact with the same checksum the artifact content is copied over to
+the new location without requiring content transfer.
+
+```python
+from artifactory import ArtifactoryPath
+
+path = ArtifactoryPath("http://my-artifactory/artifactory/my_repo/foo")
+sha1 = "1be5d2dbe52ddee96ef2d17d354e2be0a155a951"
+sha256 = "00bbf80ccca376893d60183e1a714e707fd929aea3e458f9ffda60f7ae75cc51"
+
+# If you don't know sha value, you can calculate it via
+# sha1 = artifactory.sha1sum("local_path_of_your_file")
+# or
+# sha256 = artifactory.sha256sum("local_path_of_your_file")
```

Comment:
Hi @beliaev-maksim 

How about this change?

Reply:
no need for duplication. It should be clear from functions themselves.

just uncomment these lines and remove hardcoded SHA values

Reply:
Could I know why do we guide user to calculate sha value by `artifactory.sha1sum`? What if they know sha value already?